superset-frontend/src/dashboard/components/resizable/ResizableContainer.jsx

```diff
@@ -1,327 +0,0 @@
-/**
```

Comment:
for future reference - if you rename a file using `git mv`, git should figure out that it's really the same file and show the diff correctly